lib/block-template-utils.php

```diff
@@ -95,9 +96,38 @@ function gutenberg_generate_block_templates_export_file() {
 		$theme_json_raw     = array_merge( $schema, $theme_json_raw );
 	}
 
+	// Find any uploaded files.
+	$uris_to_migrate = WP_Theme_JSON_Resolver_Gutenberg::get_migrated_relative_theme_uris(
+		$tree,
+		array(
+			'relative_path_prefix' => 'file:./assets/',
+		)
+	);
+	if ( ! empty( $uris_to_migrate ) ) {
+		$uploads = wp_upload_dir();
+		foreach ( $uris_to_migrate as $uri ) {
+			$path         = explode( '.', $uri['target'] );
+			$file         = str_replace( $uploads['baseurl'], $uploads['basedir'], $uri['name'] );
+			$file_content = file_get_contents( $file );
+			if ( ! $file_content ) {
+				continue;
+			}
+			// @TODO: Handle setting in WP_Theme_JSON_Resolver_Gutenberg, something akin to ::resolve_theme_file_uris().
```

Comment:
Ideally this would happen in the loop that checks for background images, but it could also set a flat to `'reset_image'` or something too.